nextflow_schema.json

```diff
@@ -694,6 +694,214 @@
+                },
+                "metagenomics_kraken2_save_minimizers": {
+                    "type": "boolean",
+                    "description": "Turn on saving minimizer information in the kraken2 report thus increasing to an eight column layout.",
+                    "fa_icon": "fas fa-save",
+                    "help_text": "Turn on saving minimizer information in the kraken2 report thus increasing to an eight column layout.\n\nAdds `--report-minimizer-data` to the kraken2 command."
+                },
+                "metagenomics_malt_mode": {
+                    "type": "string",
+                    "default": "BlastN",
+                    "description": "Specify which alignment mode to use for MALT. Options: 'Unknown', 'BlastN', 'BlastP', 'BlastX', 'Classifier'.",
+                    "fa_icon": "fas fa-align-left",
+                    "help_text": "Use this to run the program in 'BlastN', 'BlastP', 'BlastX' modes to align DNA\nand DNA, protein and protein, or DNA reads against protein references\nrespectively. Ensure your database matches the mode. Check the\n[MALT\nmanual](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf)\nfor more details. Default: `'BlastN'`\n\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-m`\n",
+                    "enum": ["BlastN", "BlastP", "BlastX"]
+                },
+                "metagenomics_malt_alignment_mode": {
+                    "type": "string",
+                    "default": "SemiGlobal",
+                    "description": "Specify alignment method for MALT. Options: 'Local', 'SemiGlobal'.",
+                    "fa_icon": "fas fa-align-center",
+                    "help_text": "Specify what alignment algorithm to use. Options are 'Local' or 'SemiGlobal'. Local is a BLAST like alignment, but is much slower. Semi-global alignment aligns reads end-to-end. Default: `'SemiGlobal'`\n\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-at`",
+                    "enum": ["Local", "SemiGlobal"]
+                },
+                "metagenomics_malt_min_percent_identity": {
+                    "type": "integer",
+                    "default": 85,
+                    "description": "Percent identity value threshold for MALT.",
+                    "fa_icon": "fas fa-id-card",
+                    "help_text": "Specify the minimum percent identity (or similarity) a sequence must have to the reference for it to be retained. Default is `85`\n\nOnly used when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-id`"
```

Comment:
Make sure the `Modifies` string for all parameeters all follow the 'new' format (with the list)